CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 1.8.3
+
+  - Fixes missing static folder/files
+
+## 1.8.2
+
+  - Added the migration for verbose_name changes
+
```

Comment:
This seems to be unrelated to the PR, please write only what changed in this PR.

Reply:
It seems the change log for the previous versions is missing, so I think this is an acceptable remediation, unless another maintainer wants to take care of this in a separate PR, alternatively we should just accept it.